vmware_exporter/vmware_exporter.py

```diff
@@ -64,6 +64,7 @@ def configure(self, args):
                 ignore_ssl: {3}
                 collect_only:
                     vms: True
+                    vmguests: True
```

Comment:
I need to create some tests around all this. Some of the changes we have been making, as simple as they are, scare me :/

Reply:
Yeah, tests would be awesome. I tested this as much as I could and seems to work as expected.